assets/launcher/lang/sklmessages_uk_UA.properties

```diff
@@ -123,33 +123,33 @@ manager.editor.option.directory.info.label=Деяким модифікаціям
 manager.editor.option.directory.info.link=натисніть тут.
 
 manager.editor.button.more=Більше налаштувань
-manager.editor.option.memory.name=Виділена пам'ять (RAM)
+manager.editor.option.memory.name=Виділена пам’ять (ОЗП)
```

Comment:
Because it is technical page - its better to use english abbreviations, not ukrainian ones
(and ОЗП it literally memory, not random access memory - that would be ПДД, if you didnt know)

Reply:
about this one, yep, it can be but literal only. I've never seen ПДД instead RAM or ОЗП in ukrainian translations. Nevermind.